### ISSUE TITLE
malformed package.json flashes error after each document change

### ISSUE THREAD

**Issue by tswaters** [Microsoft/vscode-eslint#127]
I encountered this creating a new package.json file in a folder with an 'app.js' - with the intent to populate it shortly... was not expecting vscode-eslint to have a total fit trying to read the blank file.

Any time any change is made to app.js, it revalidates and throws this error again & again.  I should probably fix up the package.json file, but until I do, I'm bombarded with these flashes:

>Cannot read config file: d:\Code\tests\package.json Error: Cannot read config file: d:\Code\tests\package.json Error: Unexpected end of input

Looks like the error is raised by eslint:

```
root@JUGGERNAUT:/mnt/d/tests/node_modules# grep -nR 'Cannot read config file' .
./eslint/lib/config/config-file.js:104:        e.message = "Cannot read config file: " + filePath + "\nError: " + e.message;
./eslint/lib/config/config-file.js:123:        e.message = "Cannot read config file: " + filePath + "\nError: " + e.message;
./eslint/lib/config/config-file.js:145:        e.message = "Cannot read config file: " + filePath + "\nError: " + e.message;
./eslint/lib/config/config-file.js:163:        e.message = "Cannot read config file: " + filePath + "\nError: " + e.message;
./eslint/lib/config/config-file.js:181:        e.message = "Cannot read config file: " + filePath + "\nError: " + e.message;
```

This should probably be caught and fail silently.... 

fwiw, running eslint from cli also raises this error, so it might be expected behavior of vscode-eslint to pass it along... doing nothing I suppose is acceptable.  Still, kind of annoying to see all those error flashes.

**Comment by dbaeumer** [comment]
I fixed that for the next upcoming version. The information is only showing once as a auto closing message. So no flashing and no confirmation.
